### ISSUE TITLE
Can I change WiFi chipset? 

### ISSUE THREAD

**Issue by ilseopHwang** [RT-Thread-Studio/sdk-bsp-stm32h750-realthread-artpi#228]
Hello.

Can I change WiFi chipset? If yes, can you give me quick guide?
In my study, library file in sdk-bsp-stm32h750-realthread-artpi-master\libraries\wlan_wiced_lib is from Wiced platform (Cypress).
Do I have to build a library first at Wiced platfomr?

Thanks in advance.

**Comment by Guozhanxin** [comment]
yes, you can do it. First, you can port a set of open source drivers to this development board。 refer to： https://github.com/tjwink/ART-Pi-WICED-AP6212a
